typescript/cli/src/utils/chains.ts

```diff
@@ -10,6 +10,15 @@ import { log, logRed, logTip } from '../logger.js';
 // a new chain in the list
 const NEW_CHAIN_MARKER = '__new__';
 
+// Returns a dynamic pageSize based on process.stdout.rows
+const calculatePageSize = (
```

Comment:
Recommend moving this to a util file not specific to `chains`

Reply:
move to `cli-options.ts`, which will hold functions to manipulate the cli function options